vita3k/gui/src/information_bar.cpp

```diff
@@ -313,9 +313,9 @@ static std::string get_notice_time(GuiState &gui, EmuEnvState &emuenv, const tim
     std::string date;
     const auto time_in_second = time / 1000;
     const auto diff_time = difftime(std::time(nullptr), time_in_second);
-    static const auto minute = 60;
-    static const auto hour = minute * 60;
-    static const auto day = hour * 24;
+    static constexpr auto minute = 60;
+    static constexpr auto hour = minute * 60;
+    static constexpr auto day = hour * 24;
```

Comment:
Same as before, inside a function, no need of a static for a constexpr basic type